### PR TITLE
Issue 1 user create entry

### PR DESCRIPTION
**Pull request by le-andre**
issue 1 new

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+var photoUrl = document.querySelector('#photo-url');
+photoUrl.addEventListener('input', photoFunction);
+
+function photoFunction(event) {
+  var imageInput = document.querySelector('#photo-url').value;
+  document.querySelector('.image').src = imageInput;
+  return imageInput;
```
Comment (BrettAlbright) [inline js/main.js]: you don't need this return statement as nothing is being returned

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  newObj.img = document.querySelector('#photo-url').value;
+  newObj.notes = document.querySelector('#notes').value;
+  newObj.currentEntryId = data.nextEntryId;
+  data.nextEntryId++;
+
+  data.entries.unshift(newObj);
+
+  document.querySelector('.image').reset();
+  document.querySelector('#title-name').reset();
+  document.querySelector('#notes').reset();
```
Comment (BrettAlbright) [inline js/main.js]: I would call reset on the form itself instead of calling it on the individual inputs.

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  event.preventDefault();
+}
+
+window.addEventListener('beforeunload', windowString);
+function windowString(event) {
+  var entriesJSON = JSON.stringify(data.entries);
+  localStorage.setItem('javascript-local-storage', entriesJSON);
+}
```
Comment (BrettAlbright) [inline js/main.js]: this should be in the `data.js` file. Also you should have the functionality which is loading the data from the local storage and assigning it to the `data` object when the page loads like in the javascript local storage exercise.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
A few updates needed. Make sure to reach out if you have questions, especially about the local storage functionality which should all be in `data.js`

File: js/data.js [hunk 4]
```diff
@@ -6,3 +6,18 @@ var data = {
...
+}
+
+window.addEventListener('load', windowsLoad);
+function windowsLoad(event) {
+  var previousentriesJSON = localStorage.getItem('javascript-local-storage');
+  if (previousentriesJSON !== null) {
+    data.entries = JSON.parse(previousentriesJSON);
+  }
+
+}
```
Comment (BrettAlbright) [inline js/data.js:23]: this does not need to be in a `load` event callback, just in the file is fine, it will be the first thing to run

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Just one tiny update!

**Pull request closed without merging**